.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Kleat Release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  release:
+    if: startsWith(github.repository, 'spinnaker/')
+    runs-on: ubuntu-latest
+    steps:
+        uses: actions/checkout@v2
+      - name: Unshallow
```

Comment:
I think this is equivalent to adding the following to the prior step (to get all history):
```
with:
  fetch-depth: 0
```
But then again, looks like this is what `go-releaser` has in their docs, so might be better to stay with what you have.  (Mostly just looked at this because I wasn't familiar with the `--unshallow` flag of git, which I learned today.)

Reply:
I also learned that today! I did end up going with `--unshallow` per `goreleaser` docs, but I see that `fetch-depth: 0` is what `actions/checkout` has in their docs, so no strong opinion 🤷

Reply:
No strong opinion either, fine leaving this as is! 💯